app/services/ReferenceDataService.scala

```diff
@@ -62,11 +62,7 @@ class ReferenceDataServiceImpl @Inject() (connector: ReferenceDataConnector) ext
 
   private def sort(countries: NonEmptySet[Country]): Seq[Country] =
     countries.toSeq.sortBy(
-      country =>
-        country.description match {
-          case Some(desc) => desc.toLowerCase()
-          case None       => country.code
-        }
+      country => country.description.toLowerCase
```

Comment:
I think we can remove this sort method. Sorting is done when the reference data is read in as a NonEmptySet. The ordering is defined in the Country model companion object. I think we should change this to `implicit val order: Order[Country] = (x: Country, y: Country) => x.description.compareToIgnoreCase(y.description)`